src/TestFramework/TestFramework/PublicAPI/PublicAPI.Unshipped.txt

```diff
@@ -1,2 +1,9 @@
 ﻿Microsoft.VisualStudio.TestTools.UnitTesting.DataRowAttribute.DataRowAttribute(object data) -> void
```

Comment:
move to shipped?

Reply:
Will keep it as unshipped for now and if I see the same problem of "we forgot to move it", I will revise and use only shipped file. Thanks!